### ISSUE TITLE
plotwitherror gives warnings when the error is small

### ISSUE THREAD

**Issue by martin-ueding** [HISKP-LQCD/hadron#144]
We thought that we fixed the zero-length arrow warnings in Issue #60, but this also occurs when the errors are very small. The two should both give a sensible plot:

```r
plotwitherror(0, 1, dy = 0.1, ylim = c(0, 2))
plotwitherror(0, 1, dy = 1e-10, ylim = c(0, 2))
```

The second one however gives the following warnings:

```
Warnmeldungen:
1: In arrows(x[rw], start, x[rw], end, length = arwhd.len, angle = 90,  :
  zero-length arrow is of indeterminate angle and so skipped
2: In arrows(x[rw], start, x[rw], end, length = arwhd.len, angle = 90,  :
  zero-length arrow is of indeterminate angle and so skipped
```

Apparently the length of the error needs to be sizable compared to the plot region for this to give no warning. I do not know how we could possibly know that in advance, expecially with `rep = TRUE`.

**Comment by j-ostmeyer** [comment]
Ok, we can translate lengths in coordinates into inches via
`length.in.inches = length * par("pin")[1]/diff(par("usr")[1:2])`
for x-direction and
`length.in.inches = length * par("pin")[2]/diff(par("usr")[3:4])`
for y-direction.
See details here: https://stackoverflow.com/questions/47034898/what-is-an-inch-setting-the-length-for-arrows

I'm going to implement this now.

**Comment by j-ostmeyer** [comment]
Solved by #147 .
